cmd/ocsp-responder/main.go

```diff
@@ -119,66 +121,108 @@ func makeDBSource(dbMap dbSelector, issuerCert string, log blog.Logger) (*DBSour
 	return NewSourceFromDatabase(dbMap, caCert.SubjectKeyId, log)
 }
 
+const clientName = "OCSP Responder"
```

Comment:
This isn't actually an AMQP client so let's delete this const, and instead, when logging the version string, just put a literal there. And I think in general we want to use the name of the binary. The previous "boulder-ocsp-responder" was wrong. Let's use "ocsp-responder".